src/Model/common.cpp

```diff
@@ -44,3 +44,69 @@
 bool seissol::model::testIfAcoustic(real mu) {
   return std::abs(mu) <= std::numeric_limits<real>::epsilon();
 }
+
+void seissol::model::getBondMatrix( VrtxCoords const i_normal,
+                                    VrtxCoords const i_tangent1,
+                                    VrtxCoords const i_tangent2,
+                                    real* o_N )
```

Comment:
Why did you choose the name bondMatrix? Is this commonly used? (It's a bit unclear what it actually means, so maybe a comment should be added in any case)